app/views/users/edit.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= provide(:title, "Edit") %>
+<div class="mt-4">
+   <h1 class="text-center">Edit your profile</h1>
+   <div class="container">
+       <%= form_for @user do |f| %>
+           <%= f.label :name %>
+           <%= f.text_field :name, class: "form-control" %>
+
+           <%= f.label :email %>
+           <%= f.text_field :email, class: "form-control" %>
+
+           <%= f.label :password %>
+           <%= f.password_field :password, class: "form-control" %>
+
+           <%= f.label :password_confirmation, "Confirmation" %>
+           <%= f.password_field :password_confirmation, class: "form-control" %>
+
+           <%= f.submit "Update", class: "btn btn-block btn-primary" %>
+       <% end %>
```

Comment:
please move this to a new file called `_form.html.erb` 
and you can update both `new.html.erb` and `edit.html.erb` to render a `_form.html.erb` inside it to make the file smaller and not repeat the code